packages/scss/src/components/card/mods.scss

```diff
@@ -2,12 +2,16 @@
 	--components-card-background: var(--palettes-grey-50);
 }
 
+@mixin action {
+	opacity: .5;
```

Comment:
Shouldn't we use a light version of the palette color instead of opacity?

Reply:
It was only a test that I forget to remove. 🤦